benchmarks/StiffODE/allen_cahn_wpd.jmd

```diff
@@ -0,0 +1,230 @@
+          Dict(:alg => KenCarp5(linsolve=LinSolveGMRES())),
+          Dict(:alg => ARKODE(Sundials.Implicit(), order=3, linear_solver=:GMRES)),
+          Dict(:alg => ARKODE(Sundials.Implicit(), order=4, linear_solver=:GMRES)),
+          Dict(:alg => ARKODE(Sundials.Implicit(), order=5, linear_solver=:GMRES))]
+labels = ["KenCarp3" "KenCarp4" "KenCarp5" "ARKODE3" "ARKODE4" "ARKODE5"]
+@time wp4 = WorkPrecisionSet(prob,abstols,reltols,setups;
+                            print_names=true, names=labels,
+                            numruns=5, error_estimate=:l2,
+                            save_everystep=false, appxsol=test_sol, maxiters=Int(1e5));
+
+plot(wp4, label=labels, markershape=:auto, title="IMEX methods, Krylov linsolve, medium order")
+```
+
+2.ExpRK methods
+
+```julia
+abstols = 0.1 .^ (7:11) # all fixed dt methods so these don't matter much
+reltols = 0.1 .^ (4:8)
+multipliers = 0.5 .^ (0:4)
+setups = [Dict(:alg => ETDRK3(), :dts => 1e-2 * multipliers),
+          Dict(:alg => ETDRK3(krylov=true, m=5), :dts => 1e-2 * multipliers),
+          Dict(:alg => ETDRK4(), :dts => 1e-2 * multipliers),
+          Dict(:alg => ETDRK4(krylov=true, m=5), :dts => 1e-2 * multipliers),
+          Dict(:alg => HochOst4(), :dts => 1e-2 * multipliers),
+          Dict(:alg => HochOst4(krylov=true, m=5), :dts => 1e-2 * multipliers)]
+labels = hcat("ETDRK3 (caching)", "ETDRK3 (m=5)", "ETDRK4 (caching)",
+              "ETDRK4 (m=5)", "HochOst4 (caching)", "HochOst4 (m=5)")
+@time wp5 = WorkPrecisionSet(prob,abstols,reltols,setups;
+                            print_names=true, names=labels,
+                            numruns=5, error_estimate=:l2,
+                            save_everystep=false, appxsol=test_sol, maxiters=Int(1e5));
+
+plot(wp5, label=labels, markershape=:auto, title="ExpRK methods, medium order")
+```
+
+## Between family comparisons
+
+```julia
+abstols = 0.1 .^ (7:11)
+reltols = 0.1 .^ (4:8)
+multipliers = 0.5 .^ (0:4)
+setups = [Dict(:alg => KenCarp5()),
+          Dict(:alg => ARKODE(Sundials.Implicit(), order=5, linear_solver=:Dense)),
+          Dict(:alg => KenCarp5(linsolve=LinSolveGMRES())),
+          Dict(:alg => ARKODE(Sundials.Implicit(), order=5, linear_solver=:GMRES)),
+          Dict(:alg => ETDRK3(krylov=true, m=5), :dts => 1e-2 * multipliers),
+          Dict(:alg => ETDRK4(krylov=true, m=5), :dts => 1e-2 * multipliers)]
+labels = hcat("KenCarp5 (dense linsolve)", "ARKODE (dense linsolve)", "KenCarp5 (Krylov linsolve)",
+              "ARKODE (Krylov linsolve)", "ETDRK3 (m=5)", "ETDRK4 (m=5)")
+@time wp6 = WorkPrecisionSet(prob,abstols,reltols,setups;
+                            print_names=true, names=labels,
+                            numruns=5, error_estimate=:l2,
+                            save_everystep=false, appxsol=test_sol, maxiters=Int(1e5));
+
+plot(wp6, label=labels, markershape=:auto, title="Between family, medium order")
+```
+
+## Environment information
+
```

Comment:
See the other .jmds for the updated way to do this.

Reply:
Do u mean put this instead?
```julia{echo=false}
using DiffEqBenchmarks
DiffEqBenchmarks.bench_footer(WEAVE_ARGS[:folder],WEAVE_ARGS[:file])
```

Reply:
yup